apacheconfig/wloader.py

```diff
@@ -172,5 +174,6 @@ def dump(self):
     def __str__(self):
         return ("%s(%s)"
                 % (self.__class__.__name__,
-                   str([self._type] +
+                   six.text_type(
```

Comment:
I have a reservation here... With Py2, there is this `__unicode__` built-in as well as `__str__`. I suppose we should always return `str` from `__str__` and `unicode` from `__unicode__` (where it makes sense at all).

WDYT?